contracts/mocks/VRF/VRFConsumerMock.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Very basic mockup of the VRF call. Don't use this (!) it's just for testing.

Make sure to test VRF in a testnet.